docs/use-the-network/run-a-network-server/run-console/run-console.mdx

```diff
@@ -0,0 +1,187 @@
+---
+id: run-console
+hide_title: true
+sidebar_label: Run Console
+slug: /use-the-network/run-console
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# What is Console?
+
+[Console](https://github.com/helium/console) is a project developed by Helium.
+It provides a web browser front-end for users to manage organizations and
+devices and a Helium Blockchain-compatible LoRaWAN Network Server (LNS) called
+[Router](https://github.com/helium/routerv3).
+
+We are working on integrations with third-party LNS's such as Chirpstack, but at
+this time, this is the only "plug-and-play" solution for operating an OUI. That
+being said, the Router code is open-source and custom LNS implementations may be
+derived by anyone already since OUI operation on the Helium Network is entirely
+permissionless.
+
+## Requirements
+
+Before we being, you should have the following:
+
+- [purchased an OUI](/use-the-network/buy-an-oui) and noted its number
+- installed Docker
+- you have some spare HNT to burn ($10-100 worth), for funding your OUI address
+- an Auth0 account for user account management
+
+Optionally, you should also:
+
+- have a fully sync'ed Miner or ETL instance to extract a snapshot from
+- be able to configure ports on a firewall
+
+## Console Setup
+
+### Clone the console repo
+
+```bash
+git clone https://github.com/helium/console.git
+```
+
+### Customize Console environment
+
+```
+cp .env-template .env
+```
+
+Then edit `.env`. Here's an example of what it should look like:
+
+```
+SECRET_KEY_BASE=Rrb2Tz4/HOGtez3X81/5uGsdfxiwwbIesjiQBNkIcwafpXTHJoAPEHQk0Yhw4Nuc
+CLOAK_SECRET_KEY=eAqx+rwji6pioOyanpzfxQH8cCS/M3vQjdQ5LeDcSqQ=
+GUARDIAN_SECRET_KEY=Rrb2Tz4/FOGtez3X81/5uGsdfxiwwbIesjiQBNkIcwafpXTHJoAPEHQk0Yhw4Nuc
+ROUTER_SECRETS=1524243720:Rrb2Tz4/HOGtez4X81/5uGsffxiwwbIesjiQBNkIcwafpXTHJoAPEHQk0Yhw4Nuc
+OUI=4
+SELF_HOSTED=true
+ENV_DOMAIN=localhost:4000
+AUTH_0_CLIENT_ID=NyfDZUfyT2thF7G56oazCsdGxO8GRZD6
+AUTH_0_DOMAIN=console.us.auth0.com
```

Comment:
having AUTH_0_DOMAIN and AUTH0_BASE_URL be different domains is confusing here

Reply:
good catch. i'll revert this to match the screenshots